src/main/ansible/vagrant.yml

```diff
@@ -119,18 +130,47 @@
       solr_version: "6.6.1"
       solr_package: "http://maven-repo.dans.knaw.nl/ext-release-local/org/apache/solr/{{ solr_version }}/solr-{{ solr_version }}.tgz"
 
-  # TODO fix problem that gives itself away with 'attr_' prefix for field names in the index
-  - name: Copy schemas
+  - name: Copy solr core configuration
     copy:
-      src: "../assembly/dist/install/{{ item.name }}/schema.xml"
-      dest: "{{ solr_var_dir }}/solr-home/data/{{ item.name }}/conf/"
-    with_items: "{{ solr_cores }}"
+      src: "{{ solr_core_src }}/{{ item }}"
+      dest: "{{ solr_core_dest }}/"
+    with_items:
+      - "schema.xml"
+      - "solrconfig.xml"
+
+  - name: Remove (disable) managed schemas
+    file:
+      path: "{{ solr_core_dest }}/managed-schema"
+      state: absent
 
   - service:
       name: "solr"
       state: restarted
       enabled: yes
 
+########### ldap
+
+- hosts: "test"
+  vars:
+    shared_ldap_admin_user: "cn=ldapadmin,dc=dans,dc=knaw,dc=nl"
+    shared_ldap_url: "ldap://localhost"
+    easy_ldap_dir_dans_schema_location: "/etc/openldap/slapd.d/cn=config/cn=schema/cn={12}dans.ldif"
+    easy_ldap_dir_easy_schema_location: "/etc/openldap/slapd.d/cn=config/cn=schema/cn={13}easy.ldif"
+    easy_ldap_dir_easyadmin_email: "noreply@whatever.dans.knaw.nl"
+    easy_ldap_dir_easyadmin_password: "easyadmin"
+    easy_ldap_dir_ldapadmin_password: "ldapadmin"
+    easy_test_users_ldapadmin_password: "ldapadmin"
```

Comment:
@janvanmansum 
Don't understand much of it, just grabbed something from different places. Do I need all the variables? ApacheDirectoryStudio does show the users with the above email address for easyadmin, so it isn't talking to some other instance.